### ISSUE TITLE
Acessibility options possible?

### ISSUE THREAD

**Issue by AeliusSaionji** [Shadnix-was-taken/BeatSaber-SaberTailor#22]
Hello!

Thanks to this mod I was able to make beatsaber respond to the vive tracker at an angle usable to me in this setup.

![2020-09-24 15 05 29](https://user-images.githubusercontent.com/4342746/94188594-9ace5c00-fe77-11ea-8f8b-ac5c1ca73d5d.jpg)

Something I'm struggling with as I try and advance to expert+ songs: I don't have a wrist, and I humanly cannot move my arm fast enough to reach notes, which normally are meant to be hit with the flick of the wrist.

I understand if this is not within the scope of this mod, but I figured I'd ask anyway: would it be possible to dynamically adjust the skew of the beam as I move my arm?
Here's a basic 2D mockup of what I mean:
![image](https://user-images.githubusercontent.com/4342746/94189473-c69e1180-fe78-11ea-9f48-64237274eca5.png)

If this isn't in the scope of this project, could you point me in the direction of where I should be looking?
Thanks!

**Comment by AeliusSaionji** [comment]
Steven over on the bsmg discord suggested that it could be better to incorporate velocity, too, so the angle changes are less exaggerated when I'm moving slower.

**Comment by Shadnix-was-taken** [comment]
Hi. That is an interesting use case I've never thought of, interesting.

I think I have an idea how to implement this, so I'm willing to give it a try. It'll probably take a couple of days, because work is keeping me busy at the moment. 

Enabling something like this will have to disable score submission, though. But from what I've read from your discussion in BSMG, it seems you aware of and ok with this caveat.

**Comment by AeliusSaionji** [comment]
Thank you!

I don't know how much of the BSMG backlog you read, but I realized I left out some information here:

- It would be most broadly useful to an accessible audience if configurable per hand. I only need it for my left hand, but perhaps someone else might want to use it for both hands, and have different configurations per hand.
- If possible, I would like to be able to configure the following: `maximum angle`, `deadzone`, and maybe `acceleration factor`? Or, whatever tunables make sense for how you intend to implement this.
- It also occurred to me that for 90 and 360 degree map support, the mod would have to be aware of the currently active lane. This is low priority- unless trivial, I wouldn't put effort into supporting 90/360 until we've got a working proof of concept.

Thanks again!

**Comment by Shadnix-was-taken** [comment]
Hi. Just wanted to say: I haven't forgotten about this! 
Beat Saber got a major update, which needed to be dealt with first. I'll go back to this ticket once the inital update for BS 1.12.2 is done and released.

**Comment by AeliusSaionji** [comment]
No worries!

Plus, I also need the mod as is to play at all 😆 since beatsaber by default detects the tracker as coming out at a 90 degree angle from my arm.
Now I know to hold off on updating my game 😄
